.github/workflows/e2e-tests.yml

```
@@ -81,8 +81,9 @@ jobs:
         jq --version
 
     - name: Build Production Jetpack
-      working-directory: projects/plugins/jetpack
-      run: yarn build-production-concurrently
+      run: |
+        yarn install
+        ./node_modules/.bin/jetpack build plugins/jetpack -v --production
```

This probably wouldn't run into the same problem as #19275, but it's still probably safer to do
```suggestion
        yarn jetpack build plugins/jetpack -v --production
```